images/checkout.sh

```diff
@@ -16,7 +16,10 @@ set -xe
 
 SRC_DIR=$1
 
-mkdir -p /src/${REPO_OWNER}
```

Comment:
do you need to mkdir for {SRC_DIR}/${REPO_OWNER}?

Reply:
Done

Reply:
I looked at git clone and it seems to create any nested directories during clone by itself.

`git clone <source> a/b/c/d` would create directory structure `a/b/c/d` if not already present